frontend/src/components/contactsList.tsx

```diff
@@ -50,39 +79,129 @@ export const ContactsList = (): JSX.Element => {
     getContacts().catch(console.error)
   }, [])
 
+  const toggleModal = () => {
+    setModal(!modal)
+  }
+
+  if (modal) {
+    document.body.classList.add('active-modal')
+  } else {
+    document.body.classList.remove('active-modal')
+  }
+
   return (
-    <div>
-      <h1>Contacts List</h1>
-      <button onClick={toggleContactForm}>Add Contact</button>
-      {showCreateAContactForm && <CreateContactForm setShowCreateAContactForm={setShowCreateAContactForm} handleContactCreate={handleContactCreate} />}
-      <ul>
-        {contacts?.map((contact) => (
-          <li key={contact.id}>
-            <p>{contact.firstName} {contact.lastName}</p>
-            <p>{contact.email}</p>
-            <p>{contact.phoneNumber}</p>
-            {contact.relationships !== null && (
-              <ul>
-                {contact.relationships?.map((relationship) => (
-                  <li key={relationship.id}>
-                    <p>{relationship.relationshipType}</p>
-                  </li>
-                ))}
-              </ul>
-            )}
-            {(contact.importantDateEvent !== null) && (
-              <ul>
-                {contact.importantDateEvent?.map((importantDateEvent) => (
-                  <li key={importantDateEvent.id}>
-                    <p>{importantDateEvent.date}</p>
-                    <p>{importantDateEvent.eventType}</p>
-                  </li>
-                ))}
-              </ul>
+    <div className='background'>
+      <div className="contactsHeader">
+          <img className='burgerIcon' src={burgerIcon} alt="burgerIcon" />
+          <div className='iconAndTitle'>
+            <img className='contactIcon' src={contactIcon} alt="contactIcon" />
+            <h2>Contacts</h2>
+          </div>
+          <div className="searchBar">
+            <input type="text"
+              placeholder="Search"
+              className='searchInput'
+            />
+            <FontAwesomeIcon className='magIcon' icon={faMagnifyingGlass} />
+          </div>
+      </div>
+
+      <div className="contactList">
+        <div className="leftBar">
+          <div className="createContactButton">
+            <button
+            onClick={toggleModal}
+            className="btn-modal">
+            Create contact
+            </button>
+            <img className='fourColorPlusSign' src={plusSign} alt="plusSign" />
+            {modal && (
+              <div className="modal">
+                <div onClick={toggleModal} className="overlay"></div>
+                  <div className="modal-content">
+                    <CreateContactForm handleContactCreate={handleContactCreate} />
+                    <button
+                    className='close-modal buttonStyle'
+                    onClick={toggleModal}
```

Comment:
This does this close modal button also resets the form?

Reply:
yessir